configs/gunicorn.py

```diff
@@ -0,0 +1,37 @@
+from os import getenv
+from multiprocessing import cpu_count
+
+bind = f"0.0.0.0:{getenv('APP_PORT', '8000')}"
+
+# Log section
+if getenv('DEBUG') == "true":
+    loglevel = 'debug'
+else:
+    loglevel = 'info'
+
+accesslog = '-'
+errorlog = '-'
+access_log_format = '%(h)s %(l)s %(u)s %(t)s \"%(r)s\" %(s)s %(b)s \"%(f)s\" \"%(a)s\" %({X_Forwarded_For}i)s"'
+
+# How long to keep connection alive, this should be timed
+# a little less than the ELB idle timeout
+keepalive = getenv('APP_GUNICORN_TIMEOUT', '118')
+
+# NOTE: use keepalive instead of timeout
+# timout = getenv('APP_GUNICORN_TIMEOUT', '118')
+
+if getenv('DJANGO_ENV') == 'dev':
+    workers = '3'
+else:
+    # Automatically figure number of workers if environment is not set
+    cpu = (cpu_count() * 2 + 1)
+    workers = getenv('APP_GUNICORN_WORKERS', cpu)
```

Comment:
Similarly, should this be a string (which it would be if it's in the env as `APP_GUNICORN_WORKERS`) or an int?

Reply:
Same as above and it didn't seem to really care about it being an int or string